pages/settings.js

```diff
@@ -163,6 +165,13 @@ export default function Settings ({ ssrData }) {
             items={SUPPORTED_CURRENCIES}
             required
           />
+          <Input
+            label='default max fee for withdrawals'
+            name='withdrawMaxFeeDefault'
+            groupClassName='mb-0'
```

Comment:
There is no visual separation to the next item:

![2023-10-15-200820_723x101_scrot](https://github.com/stackernews/stacker.news/assets/27162016/bd46fbf5-ff42-456c-a961-fd38b2a211f8)

This suggestion fixes that:

```suggestion
```

Including this fix, the settings page looks like this:

![2023-10-15-201132_1920x1080_scrot](https://github.com/stackernews/stacker.news/assets/27162016/fd538c2d-56f7-496e-96a2-ea1e595a2959)

I am wondering if we should create a new section for zap default, fiat currency and default max fee for withdrawals? Starts to look a bit crowded